enos/modules/vault_verify_performance_replication/scripts/verify-performance-replication.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -e
+
+binpath=${vault_install_dir}/vault
+
+fail() {
+  echo "$1" 2>&1
+  return 1
+}
+
+retry() {
+  local retries=$1
+  shift
+  local count=0
+
+  until "$@"; do
+    exit=$?
+    wait=$((10 ** count))
+    count=$((count + 1))
+    if [ "$count" -lt "$retries" ]; then
+      sleep "$wait"
+    else
+      return "$exit"
+    fi
+  done
+
+  return 0
+}
+
+check_pr_status() {
+  pr_status=$($binpath read -format=json sys/replication/performance/status)
+  cluster_state=$($binpath read -format=json sys/replication/performance/status | jq -r '.data.state')
+
+  if [[ "$cluster_state" == 'idle' ]]; then
+    fail "expected cluster state to be not idle, got $($binpath read -format=json sys/replication/performance/status | jq -r '.data.state')"
```

Comment:
I'm not sure this makes sense. We know the cluster state is idle so why are we querying for it again?

Reply:
Updated to display the `cluster_state`